src/styled/progress.ts

```diff
@@ -1,15 +1,6 @@
 // 3pp
 const cliProgress = require('cli-progress')
-const enum BarType {
-  SingleBar = 'SingleBar',
-  MultiBar = 'MultiBar'
-}
-export default function progress(type: string, settings: object) {
-  if (type === BarType.MultiBar) {
-    return new cliProgress.MultiBar(settings)
-  }
-  if (type === BarType.SingleBar) {
-    return new cliProgress.SingleBar(settings)
-  }
-  throw new Error(`${type} is not a valid progress bar type, use 'SingleBar', or 'MultiBar'`)
+
+export default function progress(settings?: any): any {
+  return new cliProgress.SingleBar(settings)
```

Comment:
We should provide default settings in this factory unless you're wanting to use cli-progress' standard ones.

Also, we need to do TTY detection and turn it off with `noTTYOutput`.